Source/WebKit/WebProcess/cocoa/RemoteRealtimeMediaSourceProxy.cpp

```diff
@@ -43,7 +43,7 @@
 namespace WebKit {
 using namespace WebCore;
 
-static IPC::Connection& getSourceConnection(bool shouldCaptureInGPUProcess)
+static Ref<IPC::Connection> getSourceConnection(bool shouldCaptureInGPUProcess)
```

Comment:
We can do this. Alternatively, the caller of this function can store IPC::Connection in a Ref/RefPtr explicitly.